core/src/main/java/org/fao/geonet/kernel/metadata/draft/DraftMetadataIndexer.java

```diff
@@ -0,0 +1,820 @@
+                        .valueOf(fullMd.getDataInfo().getRating());
+                final String displayOrder = fullMd.getDataInfo()
+                        .getDisplayOrder() == null ? null
+                                : String.valueOf(
+                                        fullMd.getDataInfo().getDisplayOrder());
+
+                if (Log.isDebugEnabled(Geonet.DATA_MANAGER)) {
+                    Log.debug(Geonet.DATA_MANAGER,
+                            "record schema (" + schema + ")"); // DEBUG
+                    Log.debug(Geonet.DATA_MANAGER,
+                            "record createDate (" + createDate + ")"); // DEBUG
+                }
+
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.ROOT, root, true, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.SCHEMA, schema, true, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.DATABASE_CREATE_DATE, createDate,
+                        true, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.DATABASE_CHANGE_DATE, changeDate,
+                        true, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.SOURCE, source, true, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.IS_TEMPLATE,
+                        metadataType.codeString, true, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.UUID, uuid, true, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.IS_HARVESTED, isHarvested, true,
+                        true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.OWNER, owner, true, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.DUMMY, "0", false, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.POPULARITY, popularity, true,
+                        true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.RATING, rating, true, true));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.DISPLAY_ORDER, displayOrder,
+                        true, false));
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.EXTRA, extra, false, true));
+                
+                //Mark as draft
+                moreFields.add(SearchManager.makeField(
+                        Geonet.IndexFieldNames.DRAFT, "Y", true, true));
+
```

Comment:
Lowercase "y" would be better and consistent with all other "boolean" properties we have eg. has_atom